src/main-output.cpp

```diff
@@ -109,7 +110,17 @@ bool main_output_is_supported()
 	auto output_groups = config->OutputGroups;
 
 	obs_data_t *output_settings = obs_data_create();
-	obs_data_set_string(output_settings, "ndi_name", "NDI Output Support Test");
+
+	// Append a random number to the test NDI name to avoid
+	// conflicts with existing outputs
+	std::random_device rd;
+	std::mt19937 gen(rd());
+	std::uniform_int_distribution<> dis(10000, 99999);
+
+	int randomNum = dis(gen);
+	std::string testString = "NDI Output Support Test " + std::to_string(randomNum);
```

Comment:
This is a nitpick...

I agree that `std::string` is "better" than `QString`, but our code almost exclusively uses QString almost everywhere, so I would prefer to be consistent.

https://github.com/search?q=repo%3ADistroAV%2FDistroAV+QString&type=code
98 uses.

https://github.com/search?q=repo%3ADistroAV%2FDistroAV%20std%3A%3Astring&type=code
37 uses.
28 of them in `obs-support`, which is technically not our code (copied from OBS internal/private code).
5 of them in our `ndi-finder` code that I guess I let slip through the cracks.
1 that calls that `obs-support` code.
1 that calls our `ndi-finder` code.
1 a map in `config.cpp` that I guess I also let slip through the cracks.
1 a map in `ndi-source.cpp` that I guess I also let slip through the cracks.